netlogo-gui/src/main/agent/LinkManager.scala

```diff
@@ -216,23 +123,23 @@ class LinkManagerImpl(world: World, linkFactory: LinkFactory) extends LinkManage
   }
 
   def addLink(link: Link) = {
-    allLinks.addLink(link)
     if (link.getBreed eq world.links) {
+      // FIXME: Wait, does this make sense? `isDirectedLink` is calculated from the breed...
```

Comment:
Can you explain this comment? It definitely needs to be removed before merge. Are you confident that this code is correct?